### ISSUE TITLE
Move model_tid and media_use_tid from global configuration options to CSV values

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_workbench#12]

**Comment by seth-shaw-unlv** [comment]
Related, we need some way to configure the proper term ID in the UI since they are variable from one instance to another.

**Comment by mjordan** [comment]
Yes, I suspect there is a REST call to get all terms in a vocab. I'll look around.

**Comment by mjordan** [comment]
As of 268fc43db4ba76a50ae204ff72b5cf33b057a26f, `field_model` is now defined in theh CSV file and not in config.

**Comment by mjordan** [comment]
> Related, we need some way to configure the proper term ID in the UI since they are variable from one instance to another.

@seth-shaw-unlv the View installed by Islandora Workbench Integration gives you all terms in a taxonomy, from where you can pick out names. Example Python code is at https://github.com/mjordan/islandora_workbench/blob/master/workbench_utils.py#L889.
